File: 2019/solutions/prob04.java
import java.util.Scanner;

public class prob04 {
	public static double WATER_WEIGHT = 62.4;
	
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		double force = 0;
		double boatVolume = input.nextDouble();
		double boatWeight = input.nextDouble(); 
		double buoyancy = input.nextDouble();
		input.close();
		force = boatVolume * buoyancy * WATER_WEIGHT;
		if(force >= boatWeight)
			System.out.println("safe");
		else
			System.out.println("heavy");
	}
}

File: 2019/solutions/prob04.js
const readline = require('readline');

const readInterface = readline.createInterface({
    input: process.stdin,
    console: false
});

readInterface.on('line', function(line) {
    const water_weight = 62.4;
    let [volume, weight, buoyancy] = line.split(' ').map((val) => {return val * 1.0});
    let supported = volume * buoyancy * water_weight;
    if(supported > weight) {
        console.log('safe');
    } else {
        console.log('heavy');
    }
});
